### ISSUE TITLE
Ability for moderators to add short publicly visible explanations to some users

### ISSUE THREAD

**Issue by ghost** [bethylamine/soupcan#68]
There are some users that are confusing/give off mixed signals that may lead users to falsely report or appeal them, e.g.:

1. someone who pretends to be trans-friendly in their bio, but is actually transphobic
2. someone who pretends to be transphobic in their bio, but is actually trans-friendly
3. someone who pretends to be transphobic in their bio, but is actually transphobic for other reasons

It might be a good idea to have an ability to add some sort of public short note to these users to avoid false reports/appeals, as it may require extensive research from someone who is not already familiar with the user to determine what the correct label is.

Yes, these have all happened (actually number 2 turned into number 3), though I have not yet seen people falsely reporting/appealing due to this, but I did almost falsely report number 2 before.

Maybe these could be also only shown when someone tries to report/appeal the user, to reduce clutter and/or giving transphobes more fuel with public notes like that directly visible on their profiles if they install the extension.

**Comment by CJH87** [comment]
I had not considered number 2 existing. I shall update my MO.

**Comment by 0xallie** [comment]
Yeah, I noticed it from their tweets and being followed by many trans people I follow. That particular person did actually start spewing transmed/TERF rhetoric recently, doesn't mean all of them will though.

**Comment by CJH87** [comment]
Ah, do you mean accounts like [@maleallyman1](https://twitter.com/maleallyman1)?

**Comment by bethylamine** [comment]
Sort of like a community notes feature? So, in the case where a user is reported only by users, the reasons are kept private, but moderators have a way to add an explainer to any red user?

**Comment by 0xallie** [comment]
Yeah, an optional note, not exactly corresponding to report reasons or anything. It could also be useful on non-red users (see case 2 in the OP), maybe it can require them to be either red or appealed if that's easier to implement?
